### DISCUSSION TITLE
XTTSv3 Development

### DISCUSSION THREAD

**Discussion by ElliottDyson** [coqui-ai/TTS#3757] [General Q&A]
Hello, just curious as to where the development of the xtts model occurs as I'd very much like to contribute to the next model that is planned to be released to aid in the betterment of open source tts.

Thanks

**Comment by eginhard** [comment]
The company shut down so the model won't be updated.

**Reply by ElliottDyson** [reply]
> The company shut down so the model won't be updated.

That's really sad to hear. Have their training data and model released at all given they were open source, so someone may be able to potentially continue their work?

**Reply by Jamestec9** [reply]
I was curious about the same thing
